### DISCUSSION TITLE
Is there a way to limit CPU cores on the Windows standalone Milvus?

### DISCUSSION THREAD

**Discussion by estebanpw** [milvus-io/milvus#17471] [Q&A and General discussion] [answered]
Hello,

I am using MILVUS on windows as described in the guide [run milvus 2.0 on windows](https://milvus.io/blog/2021-11-19-run-milvus-2.0-on-windows.md?page=1#all) using the binary package. I have noticed that the number of cores in use depend on certain things such as `nq`, `nprobe`, etc. Is it possible to limit this somehow on Windows? (I know it is possible when running milvus with docker). My use case would be to run the milvus server with e.g. only 4 cores.

Thanks,
Esteban

**Comment by yhmo** [comment]
I have no idea about this question. 
@matrixji Do you know how to limit cpu cores on windows?

**Comment by matrixji** [comment, marked as answer]
We using bat command to start milvus.exe like:

```
cd /d %~dp0
bin\milvus.exe run standalone
```

And for Windows, the `start` command could set an affinity mask to bind with CPU cores

For your instance limit with 4 cores like:
```
cd /d %~dp0
start /affinity 7 bin\milvus.exe run standalone
```
Here mask with 7 ->  01111

**Reply by estebanpw** [reply]
Thank you very much @matrixji and @yhmo
Bests
